### PR TITLE
GuiVM documentation

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: user/advanced-topics/guivm.md
Comment (marmarek) [inline user/advanced-topics/guivm.md:11]: When mentioning the change to `ro` admin rights, this should at least mention in which file to make the change. Digging through Salt documentation is not easy...

File: user/advanced-topics/guivm.md
Comment (marmarek) [inline user/advanced-topics/guivm.md:18]: "compromise", not "compromised"

File: user/advanced-topics/guivm.md
Comment (marmarek) [inline user/advanced-topics/guivm.md:40]: I'd add the session type option is in the top right corner, it isn't very obvious otherwise.

File: user/advanced-topics/guivm.md
Comment (marmarek) [inline user/advanced-topics/guivm.md:67]: There is one more step needed: attaching the actual GPU to `sys-gui-gpu`. This can be done either manually via `qvm-pci` (remember to enable permissive option), or via `qubesctl state.sls qvm.sys-gui-gpu-attach-gpu`. The latter option assumes Intel graphics card (it has hardcoded PCI address...).

File: user/advanced-topics/guivm.md
Comment (marmarek) [inline user/advanced-topics/guivm.md:73]: The user password situation is weird. Until solved, please add a link to https://github.com/QubesOS/qubes-issues/issues/6740

File: user/advanced-topics/guivm.md
Comment (marmarek) [inline user/advanced-topics/guivm.md:108]: This warning should be way more scary. This setup basically allows anyone who can reach the VNC, to take over the control. I think you can connect a second client even if a connection is active already. And even if not, you could get disconnected (even by some unrelated network issues).
Generally, this VNC service must be protected with some other (cryptographic) layer like VPN, if exposed to open network in any way. The setup as is, is useful only for purely testing machine.

File: user/advanced-topics/guivm.md
Comment (marmarek) [inline user/advanced-topics/guivm.md:110]: I'm not sure where to put it, but https://github.com/QubesOS/qubes-issues/issues/5804 should be linked as a step required on the first use (to be fixed at some point...).

And generally, looking for [issues with `C: gui-domain` label](https://github.com/QubesOS/qubes-issues/issues?q=is%3Aopen+is%3Aissue+label%3A%22C%3A+gui-domain%22) is a good idea in case of troubles.

**Review by marmarek — CHANGES_REQUESTED** [review]

File: user/advanced-topics/guivm.md
Comment (fepitre) [reply user/advanced-topics/guivm.md:110]: Yes, I've it on my TODO list to possibly add `--all` options to `qvm-appmenus` to ease creation/update notably after switching to GuiVM.

File: user/advanced-topics/guivm.md
Comment (fepitre) [reply user/advanced-topics/guivm.md:67]: Yes, that's what I forgot to write when I tested it yesterday. Thank you for having noticed it too.

File: user/advanced-topics/guivm.md
Comment (fepitre) [reply user/advanced-topics/guivm.md:40]: I will add a screenshot.

File: user/advanced-topics/guivm.md
Comment (fepitre) [reply user/advanced-topics/guivm.md:18]: Yeah 🤣  sounds horrible

File: user/advanced-topics/guivm.md
Comment (fepitre) [reply user/advanced-topics/guivm.md:11]: Sure, that's on the improvement to be done.

File: user/advanced-topics/guivm.md
Comment (fepitre) [reply user/advanced-topics/guivm.md:108]: yes this is the current state of where I am for this :)

**Comment by fepitre** [comment]
PipelineRetry

**Comment by tlaurion** [comment]
Line 77: permission -> permissive

**Comment by fepitre** [comment]
@andrewdavidwong @marmarek  anything else to add?

**Comment by andrewdavidwong** [comment]
Looks like it's waiting for approval from Marek, then for the stable 4.1 release.

**Comment by andrewdavidwong** [comment]
@fepitre, could you please convert to relative links?

https://www.qubes-os.org/doc/documentation-style-guide/#relative-vs-absolute-links

**Comment by fepitre** [comment]
> @fepitre, could you please convert to relative links?
> 
> https://www.qubes-os.org/doc/documentation-style-guide/#relative-vs-absolute-links

Done. Thank you.

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of 777e1bccbbf05e3248077478c77829f7a1221f0e
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmIAJNwACgkQ8yiUvpaE
k4rTMA//akt9/LG6aDXpulXJp/J+Ejwhyi3HcVXM7DAHKaegcQdWKrlsas6suQWe
MsA6vm7bYU+OGbkXORHeWVvVJgWqa+ULrSJ0EUjk3dwb0y0AvDvM9aGNGUlmXHMb
snnWuz8aX7IBl78s+C7Znoh7ISjA6efy/sGg10d5LTIvm7d1gOtWa2FAr6koy8KW
5GZ0Fs4CMbfVqI+B04YnWWgx8mt242vV0FAYt60gqFZXlJjOGAtIjNJaxQq3Wtsh
T57ybEvtRZShNgZl4GiXF0em8ZmrcyIBKUGGCSHOU2K1Oy0TOY/az+4DchFEA+rN
ZQPPuqb5YXrFJ7kNq47/JzSB8MrbOZam/OllRVv17MydK8Ogz8ZlRnV4ox22biGi
fJIwrzga4w1FwRqiO5orAMDGdP+OPzmTeBBUVq8s5nHPXgxnJ6GjF+LR8SPRlO0E
tKTenQF/Qf2VjvLKKeh9RMjWs0byvTS6QiFZydujSCZuQD8ErWmzogSE1SEQ+Hhl
U5pyqvaGqBMocBvPN7S86n9MbrilhEDz1Rbgy1VFCHuoUPflbDFpSooAByCmBn5r
/5pzmQLqSTLKqmGIcMPsBq+s4ePiHz5QzMyijHRBe9K4tuyK3lZ2++65duMbyXYL
jEay2GdB0WtTUI3FKe20x8jbUWmu5MrlpOL/IFkrUKm5jc8wc9k=
=6JTk
-----END PGP SIGNATURE-----
```

**Pull request merged**
